extensions/pkg/controller/backupbucket/backupbucket_test.go

```diff
@@ -178,34 +178,9 @@ func runTest(c client.Client, namespaceName string, ignoreOperationAnnotation bo
 
 	var cleanupHandle framework.CleanupActionHandle
 	cleanupHandle = framework.AddCleanupAction(func() {
-		By("provoke error in deletion")
-		Expect(updateBackupBucketObject(ctx, c, backupBucket, func() {
-			metav1.SetMetaDataAnnotation(&backupBucket.ObjectMeta, test.AnnotationKeyDesiredOperationState, test.AnnotationValueDesiredOperationStateError)
-		})).To(Succeed())
-
 		By("delete backupbucket")
 		Expect(client.IgnoreNotFound(c.Delete(ctx, backupBucket))).To(Succeed())
```

Comment:
I guess, we would need to forcefully remove the finalizer here, if the test fails somewhere in the middle, in order to unblock namespace deletion.
If one executes this test against an existing cluster, it would be nice to cleanup properly.

Reply:
Done.